src/SFA.DAS.AANHub.Application/Partners/Queries/GetPartnerMemberQueryValidator.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentValidation;
+
+namespace SFA.DAS.AANHub.Application.Partners.Queries
+{
+    public class GetPartnerMemberQueryValidator : AbstractValidator<GetPartnerMemberQuery>
+    {
+        public GetPartnerMemberQueryValidator()
+        {
+            RuleFor(a => a.UserName)
+                .NotNull()
```

Comment:
Mirroring another comment that I got on my GET PR. `NotEmpty` covers `NotNull` already

Reply:
Fixed. Thanks for the tip!